### PR TITLE
Styling of some components

### PR DESCRIPTION
**Pull request by ruvmello**
This PR contains some more styling for the following components:
 - registration
 - login
 - profile
 - users

### CODE REVIEW

**Comment by ruvmello** [comment]
These are the screenshots of what it looks like now.
image
image
image
image
image

**Comment by jitsedesmet** [comment]
Very nice looking!

File: frontend/test/integration/login.test.tsx [hunk 1]
```diff
@@ -29,56 +29,56 @@ describe("Login page", () => {
...
+    it("SubmitHandler for loginForm sends post request", async () => {
+        const values: LoginValues = { username: "test@mail.com", password: "pass" };
+        const response: AxiosResponse = {
+            data: {},
+            status: StatusCodes.TEMPORARY_REDIRECT,
+            statusText: ReasonPhrases.TEMPORARY_REDIRECT,
+            headers: {},
+            config: {},
+            request: { responseURL: "/home" },
+        };
```
Comment (jitsedesmet) [inline frontend/test/integration/login.test.tsx:71]: This should be handled in the  entity provider.

**Review by jitsedesmet — COMMENTED** [review]
Seems very good to me. I will not approve this PR because it handles styling and I don't know anything about this, I think you should ask someone else.

I added one comment and would like to ask you to also use this PR to add some docs to the files/ functions it uses.

**Review by jitsedesmet — APPROVED** [review]

**Pull request merged**